### PR TITLE
chore: bump version, use standard maintainer list

### PR DESCRIPTION
**Pull request by henryiii**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,14 +17,15 @@ build:
...
-    - python >=3.6
+    - python >=3.7
   run:
-    - matplotlib-base >=3.2
+    - matplotlib-base >=3.4
     - numpy >=1.16.0
     - packaging
-    - python >=3.6
+    - python >=3.7
     - requests >=2.21,<3
```
Comment (andrzejnovak) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,14 +17,15 @@ build:
...
     - scipy >=1.1.0
```
Comment (andrzejnovak) [inline recipe/meta.yaml]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -17,14 +17,15 @@ build:
...
+    - mplhep_data
```
Comment (andrzejnovak) [inline recipe/meta.yaml:28]:
```suggestion
- uhi>=0.2.0
```

**Review by andrzejnovak — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 4]
```diff
@@ -17,15 +17,14 @@ build:
...
+    - matplotlib-base >=3.4
     - numpy >=1.16.0
     - packaging
-    - python >=3.6
-    - requests >=2.21,<3
-    - scipy >=1.1.0
-
+    - python >=3.7
+    - mplhep_data
+- uhi>=0.2.0
```
Comment (henryiii) [inline recipe/meta.yaml]:
```suggestion
    - uhi>=0.2.0
```

Comment (andrzejnovak) [inline recipe/meta.yaml]:
```suggestion
    - uhi>=0.2.0
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -17,15 +17,14 @@ build:
...
+    - matplotlib-base >=3.4
     - numpy >=1.16.0
     - packaging
-    - python >=3.6
-    - requests >=2.21,<3
-    - scipy >=1.1.0
-
+    - python >=3.7
+    - mplhep_data
+    - uhi>=0.2.0
```
Comment (henryiii) [inline recipe/meta.yaml]:
```suggestion
    - uhi >=0.2.0

```

**Comment by henryiii** [comment]
Shouldn't you should wait to see if it builds?

**Comment by henryiii** [comment]
(Admittedly, if it fails to build on master, it just won't upload, so not a huge problem)

**Comment by andrzejnovak** [comment]
I thought I saw it completed all check

**Comment by henryiii** [comment]
No, just the lint. It needed to be rerendered, as well.

**Pull request merged**